### ISSUE TITLE
Support of Remote Terminals in VSCode

### ISSUE THREAD

**Issue by oskar-gmerek** [withfig/fig#2656]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my feature

### Feature Details

# Description:
Currently, I have a setup consisting of an iMac, where I do most of my work, and a MacBook, where I keep the code I'm working on. I use VSCode Remote Tunnels to connect from the iMac to the MacBook. Additionally, both Macs have Fig installed.

The setup works well, but I've encountered an issue with Fig autocomplete. Since I'm working on the code on my MacBook while using the terminal on my iMac (connected via VSCode Remote Tunnels), when I trigger an autocomplete in the terminal, it appears in an unexpected location – not in the iMac's terminal, where I'm typing the command (cd, for example), but instead at the center of the MacBook's screen.

I believe this problem could potentially affect various other situations and tools used for remote work.

I expected there might be some challenges with Fig in such a setup, but I assumed it would mostly be related to having two instances of Fig running.

Feature Request:
I would like to request an improvement to Fig's autocomplete functionality when working with remote terminals in VSCode. It would be immensely helpful if the autocomplete suggestions appeared in the terminal window where the command is being typed, rather than the remote machine's screen.

This improvement would streamline the workflow for remote development and enhance the overall user experience. As more and more developers are adopting remote work setups, I believe this enhancement would benefit a growing number of users.

# Environment:

Operating System: macOS (iMac and MacBook)
IDE: Visual Studio Code
Terminal: VSCode Remote Tunnels
Tool: Fig (with autocomplete feature)
Reproducible Steps:

1. Set up VSCode Remote Tunnels to connect from mac1 to mac2.
2. Open a file in VSCode from the remote location (mac2).
3. Use the terminal in VSCode on the mac1 to type a command that triggers autocomplete (e.g., cd ).

# Current Behavior:
Autocomplete suggestions appear at the center of the remote machine's screen (mac2) instead of the local terminal in VSCode on the mac1 where terminal commands are typed .

# Expected Behavior:
Autocomplete suggestions should appear directly within the terminal window on the mac1 where the command is being typed.

**Comment by oskar-gmerek** [comment]
Maybe related or partialy related to #1688

**Comment by chaynabors** [comment]
Just asking, normally fig autocomplete positions itself correctly in your vscode window? A bash/zsh instance without tunneling works?

**Comment by oskar-gmerek** [comment]
@chaynabors Yes. On both devices everything works correctly without tunnelling.
